### PR TITLE
Move package/license to GeyserMC, gradle conventions and submodules

### PR DESCRIPTION
**Pull request by AlexProgrammerDE**

### CODE REVIEW

File: protocol/src/main/java/org/geysermc/mcprotocollib/network/Session.java [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
-import com.github.steveice10.packetlib.packet.Packet;
-import com.github.steveice10.packetlib.packet.PacketProtocol;
+package org.geysermc.mcprotocollib.network;
+
+import org.geysermc.mcprotocollib.network.codec.PacketCodecHelper;
+import org.geysermc.mcprotocollib.network.crypt.PacketEncryption;
+import org.geysermc.mcprotocollib.network.packet.Packet;
+import org.geysermc.mcprotocollib.network.packet.PacketProtocol;
+import org.geysermc.mcprotocollib.network.event.session.SessionEvent;
+import org.geysermc.mcprotocollib.network.event.session.SessionListener;
```
Comment (onebeastchris) [inline protocol/src/main/java/org/geysermc/mcprotocollib/network/Session.java]: Very nitpicky, but why did you change import order here? shouldn't the event imports come before the packet imports?

File: README.md [hunk 2]
```diff
@@ -4,40 +4,74 @@ MCProtocolLib is a simple library for communicating with Minecraft clients and s
 
 ## Example Code
 
-See the [example](https://github.com/GeyserMC/MCProtocolLib/tree/master/example/src/main/java/com/github/steveice10/mc/protocol/test) folder for sample usage.
+See the [example](https://github.com/GeyserMC/MCProtocolLib/tree/master/example/src/main/java/org/geysermc/mc/protocol/test) folder for sample usage.
```
Comment (onebeastchris) [inline README.md]: Link needs updating, e.g. to:
```suggestion
See the [example](https://github.com/GeyserMC/MCProtocolLib/tree/master/example/src/main/java/org/geysermc/mcprotocollib/) folder for sample usage.
```

**Review by onebeastchris — APPROVED** [review]
lgtm

**Review by Konicai — APPROVED** [review]
